invokers/java/package.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
We need this in multiple places. At first, I tried to just add a make target, but that needs the entire repo in order to resolve the includes. Extracting to a simple script was the next simplest thing.